PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-azure-sdk/src/main/java/com/microsoft/azure/toolkit/intellij/azure/sdk/buildtool/ServiceBusReceiverAsyncClientCheck.java

```diff
@@ -0,0 +1,34 @@
+package com.microsoft.azure.toolkit.intellij.azure.sdk.buildtool;
+
+import com.intellij.codeInspection.*;
```

Comment:
Where are the tests?

Reply:
We're manual testing for now -- testing Intellij frameworks isn't feasible for now:)

Reply:
I think we have different definitions of "testing". I'm referring to unit tests.  They are feasible. See: https://gist.github.com/conniey/28b8a35b1ac7a1e4930d47e086a73f34

Reply:
tests have been added:) thanks!